clang-tools-extra/test/clang-tidy/checkers/bugprone/exception-escape.cpp

```diff
@@ -586,6 +586,16 @@ void swap(int&, int&) {
   throw 1;
 }
 
+void iter_swap(int&, int&) {
+  // CHECK-MESSAGES: :[[@LINE-1]]:6: warning: an exception may be thrown in function 'iter_swap' which should not throw exceptions
+  throw 1;
+}
+
+void iter_move(int&, int&) {
```

Comment:
NOTE: Isn't iter_move taking single argument (iterator) ?

Reply:
Yeah the std:: common_iterator::iter_move

But the check checks only if the function name is iter_move then performs the check.

If you do prefer I could change it to only one arg

Reply:
One argument would be better. Just to keep it in sync with standard.
But thats not a blocker.